.travis.yml

```diff
@@ -54,7 +54,7 @@ install:
       conda install --quiet --file minimal-conda-requirements.txt;
     else
       if [[ "$TRAVIS_PYTHON_VERSION" == 3* ]]; then
-        sed -e '/ecmwf_grib/d' -e '/esmpy/d' -e 's/#.\+$//' conda-requirements.txt | xargs conda install --quiet;
+        sed -e '/esmpy/d' -e 's/#.\+$//' conda-requirements.txt | xargs conda install --quiet;
```

Comment:
this is a nasty hack to make sure `ecmwf_grib` is not installed for python3, where it won't build.  This is no longer required as `eccodes` is py3 compatible